src/utils/Models.tsx

```diff
@@ -0,0 +1,34 @@
+
+interface Mood {
+    id: string
+    mood: number
+    description: string
+    createdAt: string
+}
+
+// export interface Habit {
```

Comment:
Usually I would not leave commented out code, but I believe we are going to need this very soon.